packages/pigeon/lib/dart_generator.dart

```diff
@@ -470,15 +470,23 @@ void generateDart(DartOptions opt, Root root, StringSink sink) {
           'final Map<Object$nullTag, Object$nullTag> pigeonMap = <Object$nullTag, Object$nullTag>{};',
         );
         for (final NamedType field in klass.fields) {
+          final String nullsafe = field.type.isNullable ? '?' : '';
```

Comment:
Everywhere else in the code we use `isNullSafe` to declare operators, then switch on wether something is nullable or not.  That's reversed here which is a bit harder to follow.

The name "nullsafe" isn't helpful.

Reply:
I've made some changes about it. Could be that?